libsolidity/analysis/StaticAnalyzer.cpp

```diff
@@ -128,3 +131,38 @@ void StaticAnalyzer::warning(SourceLocation const& _location, string const& _des
 
 	m_errors.push_back(err);
 }
+
+bool StaticAnalyzer::visit(InlineAssembly const& _inlineAssembly)
+{
+	if (m_currentFunction)
```

Comment:
You have a point.  I'd followed the idiom from other functions, but I can change this if you have strong enough feelings about it.